packages/validators-util/src/isValidSsn/isValidSsn.test.tsx

```diff
@@ -0,0 +1,22 @@
+import { isValidSsn } from "./isValidSsn";
+
+describe("isExactLength", () => {
```

Comment:
Mer copypasta.

```suggestion
describe("isValidSsn", () => {
```


Men apropos navnet på denne: med unntak av `DogId` ligger validatorene med norske navn. Kan jeg foreslå `isValidFodselsnummer` i stedet for `isValidSsn`?